extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -602,4 +610,54 @@ private function logOut() {
 
 		return $ok;
 	}
+
+	private function addFanContributor() {
+		global $wgWordpressAPIUser, $wgWordpressAPIPassword, $wgDevelEnvironment;
+
+		$baseUrl = !empty( $wgDevelEnvironment ) ?
+			'http://sandbox2.fandom.wikia.com' :
+			'http://fandom.wikia.com';
+		$url = $baseUrl . '/ajax/create_user/';
+
+		$fields = [
+			'wp_login' => $this->mUser->getName(),
+			'wp_email' => $this->mUser->getEmail(),
+			'fandom_id' => $this->mUser->getId(),
+		];
+
+		$ch = curl_init();
+
+		curl_setopt( $ch, CURLOPT_URL, $url );
+		curl_setopt( $ch, CURLOPT_USERPWD, $wgWordpressAPIUser . ':' . $wgWordpressAPIPassword );
+		curl_setopt( $ch, CURLOPT_POSTFIELDS, $fields );
```

Comment:
Is CURLOPT_POST not needed if CURLOPT_POSTFIELDS is set? Does it assume you're sending a POST request anyway?